cvpy/annotation/base/Task.py

```diff
@@ -1,9 +1,20 @@
+from cvpy.base.ImageTable import ImageTable
+from cvpy.annotation.base.Project import Project
+
+
 class Task(object):
 
-    def __init__(self, task_id: str, start_image_id: str, end_image_id: str) -> None:
-        self._task_id = task_id
-        self._start_image_id = start_image_id
-        self._end_image_id = end_image_id
+    def __init__(self, start_image_id: int = None, end_image_id: int = None,
+                 image_table: ImageTable = None, project: Project = None) -> None:
```

Comment:
Are image_table and project required parameters to create a Task? If yes, they should be mandatory in the signature. 

def __init__(self, project: Project, image_table: ImageTable, start_image_id: int = None, end_image_id: int = None)

Reply:
Agree completely. Done!